### ISSUE TITLE
segfault with number fields

### ISSUE THREAD

**Issue by videlec** [Normaliz/PyNormaliz#27]
```
>>> import PyNormaliz
>>> nf = [ "a4-5a2+5", "a", "1.9021+/-0.01" ]
>>> C = PyNormaliz.NmzCone(number_field=nf, cone=[[[1],[0,1]],[[1],[-1]]])
>>> PyNormaliz.NmzResult(C, "ExtremeRays")
```

**Comment by sebasguts** [comment]
Should be fixed in `997ae34177285409c75aca4dccfafb2ec575fc0a`.

That was a tough one. Apparently, `PyList_SetItem` steals a reference, and you need to increase the reference count accordingly if you want to insert the same object into several lists.

**Comment by videlec** [comment]
This does fix all issues but #22. Thank you.
